### PR TITLE
Add language to improve best practice for customer DNS

### PR DESCRIPTION
**Pull request by apburnes**
Fixes issue(s) #494 

Proposed changes in this pull request:
- Add info on MTA-STS records
- Add our default headers included

[:sunglasses: PREVIEW](https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/18f/federalist.18f.gov/apb/update-dns-docs)

### CODE REVIEW

File: pages/documentation/included-with-federalist.md [hunk 1]
```diff
@@ -3,15 +3,18 @@ title: Included with Federalist
...
 
 ---
 
 * An automatic deployment mechanism for static site assets and [supported site engines]({{site.baseurl}}/documentation/supported-site-engines).
 * Web-based configuration and access to preview builds.
-* 18F's work to complete and maintain an agency Authority to Operate (ATO) for Federalist's hosting, leveraging the cloud.gov System Security Plan (SSP), with a short process to add new sites to the ATO documentation.
+* An agency Authority to Operate (ATO) for Federalist's hosting, leveraging the cloud.gov System Security Plan (SSP), with a short process to add new sites to the ATO documentation.
 * Premade templates for your sites, in addition to the many sites you can copy in creating your site. All templates now have Netlify CMS support built in, allowing content creators to make site changes without requiring extensive html, CSS, or YAML experience.
+* Automatically adds additional request headers to improve site security.
+  * `Strict-Transport-Security max-age=31536000; preload always;`
```
Comment (davemcorwin) [inline pages/documentation/included-with-federalist.md:16]:
```suggestion
  * `Strict-Transport-Security max-age=31536000; preload;`
```

**Review by davemcorwin — COMMENTED** [review]
This looks pretty good, forgot that we had some docs up here already. Is it worthwhile to communicate that the MTA-STS stuff if only relevant if they are actually using the domain for email?

**Comment by apburnes** [comment]
Added a bit a language to clarify it is for email... but didn't directly call it out.

File: pages/documentation/customer-responsibilities.md [hunk 2]
```diff
@@ -43,14 +43,22 @@ Federalist does not manage your domain name nor provide DNS services. To launch
...
->v=DMARC1; p=reject; pct=100; fo=1; ri=86400; rua=mailto:dmarcreports@gsa.gov,mailto:reports@dmarc.cyber.dhs.gov; ruf=mailto:dmarcfailures@gsa.gov
+_Expected DMARC record:_
+>`v=DMARC1; p=reject; pct=100; fo=1; ri=86400; rua=mailto:dmarcreports@gsa.gov,mailto:reports@dmarc.cyber.dhs.gov; ruf=mailto:dmarcfailures@gsa.gov`
 
-Expected SPF record:
->v=spf1 -all
+_Expected SPF record:_
+>`v=spf1 -all`
+
+SMTP MTA Strict Transport Security (MTA-STS) is a new standard that can enable domain names to opt into the strict transport layer security mode for email that requires public certificates and encryption. This standard may not be supported by all email providers but we encourage our customers to enable to use this standard if possible and plan on using the domain for email. See the standard ([RFC 3207](https://tools.ietf.org/html/rfc3207)).
```
Comment (davemcorwin) [inline pages/documentation/customer-responsibilities.md:55]:
```suggestion
SMTP MTA Strict Transport Security (MTA-STS) is a new standard that can enable domain names to opt into the strict transport layer security mode for email that requires public certificates and encryption. This standard may not be supported by all email providers but we encourage our customers to use this standard if possible when using the domain for email. See the standard ([RFC 3207](https://tools.ietf.org/html/rfc3207)).
```

**Review by davemcorwin — APPROVED** [review]
Sue gave the 👍

**Pull request merged**